test/filters/test_filters.cpp

```diff
@@ -275,6 +275,18 @@ TEST (ExtractIndices, Filters)
   */
 }
 
+TEST (PassThrough, Executor)
+{
+    PointCloud<PointXYZ> output;
+    PassThrough<PointXYZ, inline_executor<oneway_t, single_t, blocking_t::always_t>> in_pt;
+    in_pt.setInputCloud (cloud);
+    in_pt.filter (output);
+
+    PassThrough<PointXYZ, omp_executor<oneway_t, single_t, blocking_t::always_t>> omp_pt;
+    omp_pt.setInputCloud (cloud);
+    omp_pt.filter (output);
```

Comment:
Not sure why but it leads to a linker error, when the `applyFilter` function is declared out of class.

```bash
/pcl/build/../filters/include/pcl/filters/passthrough.h:199: undefined reference to `pcl::PassThrough<pcl::PointXYZ, omp_executor<oneway_t, single_t, blocking_t::always_t, std::allocator<void> > >::applyFilterIndices(std::vector<int, std::allocator<int> >&)'

```

Reply:
I have a guess, but let's address this after my comment regarding the place of executor template